runelite-api/src/main/java/net/runelite/api/events/CommandExecuted.java

```diff
@@ -26,6 +26,9 @@
 
 import lombok.Value;
 
+/**
+ * This event gets triggered whenever the player uses a command in chat like '!skill attack' or '!total'
```

Comment:
This is actually not correct. This event gets fired when a player types a `::command` into the chat.